app/assets/javascripts/discourse/app/components/search-menu-panel.hbs

```diff
@@ -1,9 +1,11 @@
 <MenuPanel
   @animationClass={{this.animationClass}}
-  @panelClass="search-menu-panel"
+  @panelClass="search-menu-panel {{if this.isEmpty 'empty'}}"
```

Comment:
we have a helper called concatClass can you try to use it here please?

Reply:
done